drivers/drivers.go

```diff
@@ -0,0 +1,11 @@
+package drivers
+
+// The Driver interface is the set of functions required to start, manage, and verify a Kubernetes cluster. Inspired by the steps used in kube-up.sh, with extensions for management and scaling of clusters.
+type Driver interface {
+	GenerateCerts()
+	GetTokens()
+	ProvisionMaster()
+	ConfigureMaster()
+	ProvisionNode()
```

Comment:
Also, going to use viper for a single config surface that cobra/env vars all pipe into (so config and cli aren't tied tightly together) for this first provider.